<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_write">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="messageText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>771</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="writeContent">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="plainText">
     <string>test write files in winrt kkadddsdaf dsdafadsfasdfs adsfasdfasdfkkkkkkk399999999asdfas9dfasdfasdf9adsfasdfasdfa3kpöiukhjjhfgjdf56lhklg5ertest write files in winrt kkadddsdaf dsdafadsfasdfs adsfasdfasdfkkkkkkk399999999asdfas9dfasdfasdf9adsfasdfasdfa3kpöiukhjjhfgjdf56lhklg5ertest write files in winrt kkadddsdaf dsdafadsfasdfs adsfasdfasdfkkkkkkk399999999asdfas9dfasdfasdf9adsfasdfasdfa3kpöiukhjjhfgjdf56lhklg5er</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="read_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openReadOnly">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read file (ReadOnly)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seek">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>seek</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pos">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>position</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_flush">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>flush</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Write content</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug Print</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Read file content</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="seek_spinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="positionText">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="writetimes">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="filtersString">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>filtername (.doc)</string>
    </property>
   </widget>
   <widget class="QLabel" name="bytesToWrite">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bytes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
